index.bs

```diff
@@ -303,6 +308,11 @@ Picture-in-Picture.
 The API will have to be used with the [[MediaSession]] API for customizing the
 available controls on the Picture-in-Picture window.
 
+The user agent SHOULD not override the <a>media session action</a>
+<a>enterpictureinpicture</a> action handler,  if one is provided by the site.
```

Comment:
Instead of `<a>`, I think bikeshed uses `<{HTMLVideoElement/enterpictureinpicture}>` or just straight up `{{HTMLVideoElement/enterpictureinpicture}}`.